caliban/cli.py

```diff
@@ -549,13 +549,15 @@ def generate_docker_args(job_mode: conf.JobMode,
   adc_loc = csdk.get_application_default_credentials_path()
   adc_path = adc_loc if os.path.isfile(adc_loc) else None
 
-  # TODO we may want to take a custom path, here, in addition to detecting it.
+  # TODO we may want to take custom paths, here, in addition to detecting them.
   reqs = "requirements.txt"
+  conda_env = "environment.yml"
```

Comment:
I followed the hardcoding template that we used for `requirements.txt` vs threading an arg through.

Reply:
seems cleaner for something like this